plugins/include/sqlx.inc

```diff
@@ -105,171 +129,248 @@ native SQL_QuoteString(Handle:db, buffer[], buflen, const string[]);
- * @param query - Handle to the query, do not free it.
- * @param error - An error message, if any.
- * @param errnum - An error code, if any.
- * @param data - Data array you passed in.
- * @param size - Size of the data array you passed in.
- * @param queuetime - Amount of gametime that passed while the query was resolving.
- * 
- * public QueryHandler(failstate, Handle:query, error[], errnum, data[], size, Float:queuetime)
- *
- * Note! The handle you pass in is a DB Tuple, NOT an active connection!
- * Note! The handle does not need to be freed.
- * Also note: This function is not guaranteed to be in another thread
- *  (in fact - it's not).  You're seeing data "after the fact", 
- *  and as such to execute another query you should run 
- *  SQL_ThreadQuery again with new data.
+ * @param db_tuple      Tuple handle, returned from SQL_MakeDbTuple().
+ * @param handler       A function to be called when the query finishes. It has to be public.
+ * @param query         The query string.
+ * @param data          Additional data array that will be passed to the handler function.
+ * @param dataSize      The size of the additional data array. 
  */
 native SQL_ThreadQuery(Handle:db_tuple, const handler[], const query[], const data[]="", dataSize=0);
 
 
 /**
- * Executes a query.
- * Returns 1 if the query succeeded.
- * Returns 0 if the query failed.
- * NOTE: You can call this multiple times as long as its parent
- *  connection is kept open.  Each time the result set will be freed
- *  from the previous call.
+ * Executes an already prepared query.
+ * 
+ * @note You can call this multiple times as long as its parent connection is kept open. 
+ *       Each time the result set from the previous call will be freed.
+ *
+ * @param query         Handle of a prepared query to be executed.
+ * 
+ * @return              1 if the query succeeded, 0 if the query failed.
  */
 native SQL_Execute(Handle:query);
 
+
 /**
  * Gets information about a failed query error.
- * Returns the errorcode.
+ *
+ * @param query         Handle of a query to extract the error from.
+ * @param error         Buffer where to store the error string.
+ * @param maxlength     The maximum length of the output buffer.
+ * 
+ * @return              The error code.
  */
 native SQL_QueryError(Handle:query, error[], maxlength);
 
 
 /**
- * Returns 1 if there are more results to be read,
- *  0 otherwise.
+ * Check whether there are more results to be read.
+ *
+ * @param query         Handle of a query to check.
+ * 
+ * @return              1 if there are more results, 0 otherwise.
  */
 native SQL_MoreResults(Handle:query);
 
 
 /**
- * Tells whether a specific column in the current row
- *  is NULL or not.
+ * Tells whether a specific column in the current row is NULL or not.
+ *
+ * @param query         Handle of a query to check.
+ * @param column        Which column to check for NULL.
+ * 
+ * @return              1 if the column is NULL, 0 otherwise.
```

Comment:
Misses `@error` with those possibles errors `Invalid query handle`, `No result set in this query`, `Invalid column`